tests/common/devices.py

```diff
@@ -312,6 +312,28 @@ def get_networking_uptime(self):
             self.logger.error("Exception raised while getting networking restart time: %s" % repr(e))
             return None
 
+    def get_image_info(self):
+        """
+        @summary: get list of images installed on the dut.
+                  return a dictionary of "current, next, installed_list"
+        """
+        output = self.command("sonic_installer list")["stdout"]
+        lines  = output.split('\n')
+        ret    = {}
+        list   = []
+        for line in lines:
+            words = line.strip().split(' ')
+            if len(words) == 2:
+                if words[0] == 'Current:':
+                    ret['current'] = words[1]
+                elif words[0] == 'Next:':
+                    ret['next'] = words[1]
```

Comment:
can use ret[words[0][:-1]] = words[1]

Reply:
Not sure I want to keep the first letter capital.
And also trying to keep the list specific due to some platform has extra output in certain versions.